bundles/audience_targeting.rst

```diff
@@ -12,62 +12,48 @@ target group of the visitor.
 Installation
 ------------
 
-Enable the bundle in `app/AbstractKernel.php`:
+Enable the bundle in `config/bundles.php`:
 
 .. code-block:: php
 
     <?php
-    abstract class AbstractKernel extends SuluKernel
-    {
-        // ...
-
-        public function reigsterBundles()
-        {
-            $bundles = [
-                // ...
-                new Sulu\Bundle\AudienceTargetingBundle\SuluAudienceTargetingBundle(),
-            ];
-
-            return $bundles;
-        }
-    }
+    Sulu\Bundle\AudienceTargetingBundle\SuluAudienceTargetingBundle::class => ['all' => true],
 
 Add the routes for the administration interface to the routing configuration
-file (`app/config/admin/routing.yml`):
+file (`config/routes/sulu_admin.yaml`):
 
 .. code-block:: yaml
 
     # ...
-    sulu_audience_targeting:
-        resource: "@SuluAudienceTargetingBundle/Resources/config/routing.xml"
-        prefix: /admin/target-groups
-
     sulu_audience_targeting_api:
-        resource: "@SuluAudienceTargetingBundle/Resources/config/routing_api.xml"
+        resource: "@SuluAudienceTargetingBundle/Resources/config/routing_api.yml"
```

Comment:
@nnatter is here `type: rest` missing? for be compatible with symfony 3.4 name_prefix?